lib/api/core/pluginsContext.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  ResponseObject: require('./models/responseObject'),
+  RealTimeResponseObject: require('./models/realTimeResponseObject'),
+
+  KuzzleError: require('./errors/kuzzleError'),
+  BadRequestError: require('./errors/badRequestError'),
+  ForbiddenError: require('./errors/forbiddenError'),
+  InternalError: require('./errors/internalError'),
+  NotFoundError: require('./errors/notFoundError'),
+  PartialError: require('./errors/partialError'),
+  ServiceUnavailableError: require('./errors/serviceUnavailableError'),
+  UnauthorizedError: require('./errors/unauthorizedError')
```

Comment:
Exactly. But this solution requires an index.js file, which need to be manually updated when a new error is updated. This may not be a problem though.

I was also wondering if we could simply list all .js files in the errors folder and require them.